src/contrib/cluster/Akka.DistributedData/ORDictionary.cs

```diff
@@ -280,7 +280,7 @@ internal ORDictionary(ORSet<TKey> keySet, IImmutableDictionary<TKey, TValue> val
 
         IReplicatedDelta IDeltaReplicatedData.Delta => Delta;
 
-        IReplicatedData IDeltaReplicatedData.MergeDelta(IReplicatedDelta delta) => Merge((IDeltaOperation)delta);
+        IReplicatedData IDeltaReplicatedData.MergeDelta(IReplicatedDelta delta) => MergeDelta((IDeltaOperation)delta);
```

Comment:
This was the fix for #4184 - needed to call `MergeDelta` here rather than `Merge`.